docs/messaging/messaging-spans.md

```diff
@@ -191,17 +191,24 @@ Messaging spans SHOULD follow the overall [guidelines for span names](https://gi
 
 <!-- markdown-link-check-disable -->
 <!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
-The **span name** SHOULD be `{messaging.operation.name} {destination}` (see below for the exact definition of the [`{destination}`](#destination-placeholder) placeholder).
+The **span name** SHOULD be `{messaging.operation.name} {destination}`
+(see below for the exact definition of the
+[`{destination}`](#destination-placeholder) placeholder).
 <!-- markdown-link-check-enable -->
 
-Semantic conventions for individual messaging systems MAY specify different span name format and then MUST document it in semantic conventions for specific messaging technologies.
+Semantic conventions for individual messaging systems MAY specify different
+span name format and then MUST document it in semantic conventions
+for specific messaging technologies.
 
-The <span id="destination-placeholder">`{destination}`</span> SHOULD describe the entity that the operation is performed against
+The <span id="destination-placeholder">`{destination}`</span>
+SHOULD describe the entity that the operation is performed against
 and SHOULD adhere to one of the following values, provided they are accessible:
 
 1. `messaging.destination.template` SHOULD be used when it is available.
 2. `messaging.destination.name` SHOULD be used when the destination is known to be neither [temporary nor anonymous](#temporary-and-anonymous-destinations).
 3. `server.address:server.port` SHOULD be used only for operations not targeting any specific destination(s).
+4. The placeholder `(temporary)` SHOULD be used when the destination is [temporary or anonymous](#temporary-and-anonymous-destinations)
```

Comment:
we can just say that destination should not be provided if there is no low-cardinality destination. 

E.g. this is how DB span name is phrased

https://github.com/open-telemetry/semantic-conventions/blob/f0974a892c308511bcfd21fc13dab2d3e598a9fa/docs/database/database-spans.md?plain=1#L68-L73

It's not blocking. 
If you believe that having `(temporary)` marker is helpful, let's go with it.

Reply:
Yeah given how the outcome now of the other PR https://github.com/open-telemetry/semantic-conventions/pull/1531 - we will have both the flag indicating it is temporary/anonymous + the actual name in the `messaging.destination.name` I don't see value in using the placeholder here.